packages/image_picker/image_picker/test/image_picker_deprecated_test.dart

```diff
@@ -240,132 +177,89 @@ void main() {
           await picker.getVideo(
               source: ImageSource.camera,
               maxDuration: const Duration(hours: 1));
-          expect(
-            log,
-            <Matcher>[
-              isMethodCall('pickVideo', arguments: <String, dynamic>{
-                'source': 0,
-                'maxDuration': null,
-                'cameraDevice': 0,
-              }),
-              isMethodCall('pickVideo', arguments: <String, dynamic>{
-                'source': 0,
-                'maxDuration': 10,
-                'cameraDevice': 0,
-              }),
-              isMethodCall('pickVideo', arguments: <String, dynamic>{
-                'source': 0,
-                'maxDuration': 60,
-                'cameraDevice': 0,
-              }),
-              isMethodCall('pickVideo', arguments: <String, dynamic>{
-                'source': 0,
-                'maxDuration': 3600,
-                'cameraDevice': 0,
-              }),
-            ],
-          );
+
+          verifyInOrder(<Object>[
+            mockPlatform.pickVideo(
+                source: ImageSource.camera,
+                preferredCameraDevice: CameraDevice.rear,
+                maxDuration: null),
+            mockPlatform.pickVideo(
+                source: ImageSource.camera,
+                preferredCameraDevice: CameraDevice.rear,
+                maxDuration: const Duration(seconds: 10)),
+          ]);
```

Comment:
You made four calls to `picker.getVideo`, but only two are verified.

Reply:
In the non-deprecated version of the test, 2 of the calls were removed. I've removed the two extra calls from this test as well, so the verifyInOrder is correct, and both tests stay "consistent" with each other.

Reply:
Thanks @ditman for pushing the fix for this :)

(I ported the main file, then copy/pasted expectations over to the deprecated version and changed the method names, but when I pasted in these expectations I forgot I'd deleted some of the tests since having different duration units isn't interesting in the context of a layer that doesn't change the type from a `Duration` object)